tag-release.sh

```diff
@@ -13,6 +13,30 @@ dep_hash() {
     git show "${MINER}:rebar.lock" | grep -A2 "${DEP}" | sed -e "s/.*\"\(.*\)\".*/\1/" | tr '\n' ' ' | awk '{print $3}'
 }
 
+strip_zeros() {
+    local VZN=$(echo $1 | sed -e 's/0//g')
+
+    if [[ $VZN == "" ]]; then
+        echo "0"
+    else
+        echo $VZN
+    fi
+}
+
+val_tag() {
+    local VAL_VERSION
+    local MAJ
+    local MIN
+    local PATCH
+
+    VAL_VERSION=$(grep -A 1 "%% MMMmmmPPPP" src/miner.erl | grep -o -E '[0-9]+')
```

Comment:
can you add a comment in `src/miner.erl` to the effect that we rely on that comment in the tagging script?

Reply:
i can do this as well, but i tried changing the pattern to match against the structure of the erlang `version/0` function itself, so as long as no one writes any comments that end the line in a period just maintaining a proper erlang function wrapped around that version number should work

Reply:
any more thoughts on this vs. updating the comment to add some weight to it @evanmcc ?